ui/src/routes/guides/a11ymenu.svelte

```diff
@@ -0,0 +1,170 @@
+<script>
+	import {toolName} from 'app/config';
+
+</script>
+
+<svelte:head>
+	<title>{toolName}: A11y menu guide</title>
+	<meta
+		content='{toolName}: A11y menu guide for this tool'
+		name='description'
+	>
+</svelte:head>
+
+<main>
+	<section>
+		<h2>What the a11y menu does</h2>
+
+		<p>
+			You can use the menu to change 2 groups of features.
+		</p>
+
+		<h3>Text features</h3>
+
+		<p>You can change:</p>
+
+		<ul>
+			<li>
+				the font (in particular we provide a font specific to help
+				people with Dyslexia),
+			</li>
+			<li>the font size,</li>
+			<li>
+				the line height (distance between lines of text in a paragraph),
+			</li>
+			<li>the letter spacing (distance between letters in a word),</li>
+			<li>the word spacing (distance between words),</li>
+		</ul>
+
+		<h3>Color features</h3>
+
+		<p>You can:</p>
+
+		<ul>
+			<li>invert all the colors in the app,</li>
+			<li>
+				select a color vision deficiency: this will remap all the colors
+				in the app to compensate for a diminished ability to see certain
+				colors,
+			</li>
+			<li>
+				edit the hue of all the colors in the app,
+			</li>
+			<li>
+				edit the saturation of all the colors in the app (by editing
+				their level of grey),
+			</li>
+			<li>edit the brightness of all the colors in the app,</li>
+			<li>edit the contrast of all the colors in the app,</li>
+		</ul>
+
+		<h2>How to use the menu</h2>
+
+		<h3>How to toggle the a11y menu on/off</h3>
+
+		<p>
+			To show or hide the a11y menu, you need to click the accessibility
+			icon in the top-right corner of the app.
+		</p>
+
+		<h3>Where to find the menu</h3>
+
+		<p>
+			On desktops, the menu should appear at the center of the screen
+			sligthly toward the bottom.
+		</p>
+
+		<p>
+			On small screens, like a mobile phone, the menu should appear below
+			the navigation header, near the thumb.
+		</p>
```

Comment:
We discussed to make this content responsive so that in `small` we see the second paragraph, while on `medium` and up we see the first paragraph.

Both paragraphs shouyld be modified not to mention the specific case we're in, see below:

`small`:

```suggestion
		<p>
			The menu should appear below
			the navigation header, near the thumb.
		</p>
```

`medium`+:

```suggestion
		<p>
			The menu should appear at the center of the screen
			sligthly toward the bottom.
		</p>
```